docs/_website/src/theme/Navbar/index.js

```diff
@@ -37,6 +37,11 @@ var items = [
     icon: "fe fe-message-square",
     label: 'Community',
   },
+  {
+    to: 'https://storybook.clutch.sh',
+    icon: "fe fe-layers",
+    label: 'Storybook',
+  },
```

Comment:
not sure we need it in the header actually, most people probably won't know what it is or be confused if they click it. better to link to it from docs pages and put it in the footer. wdyt?

Reply:
Sounds good to me